easybuild/easyblocks/o/orca.py

```diff
@@ -87,19 +87,20 @@ def install_step(self):
             ]
 
             # Version 6 extra files
-            if LooseVersion(self.version) >= LooseVersion('6.0.0'):
+            if LooseVersion(self.version) >= LooseVersion('6.1.0'):
+                files_to_copy.extend(['datasets', 'lib', (['openCOSMORS'], 'bin')])
+            else:
```

Comment:
Here the distinction between 6.1.0, 6.0.x and 5.x is lost. There should be an elif here, checking for 6.0.x. The check for greater than 5.0.0 should be after an else as it was before.